packages/next/src/server/next-server.ts

```diff
@@ -1110,8 +1110,14 @@ export default class NextNodeServer extends BaseServer<
       const normalizedReq = this.normalizeReq(req)
       const normalizedRes = this.normalizeRes(res)
 
-      const loggingFetchesConfig = this.nextConfig.logging?.fetches
-      const enabledVerboseLogging = !!loggingFetchesConfig
+      // if logging set to boolean, we handle below
+      const loggingFetchesConfig =
+        typeof this.nextConfig.logging === 'object'
```

Comment:
We should normalize next config logging into full object `{ fetches: { .. } }` for `NextConfigComplete` so that logging property will always be object with `fetches` here, instead of checking all possible input values

Reply:
if you cannot normalize this then maybe let's have a helper here to do the work determining if fetch logging is enabled